beam-ci/src/main/resources/index.jelly

```diff
@@ -0,0 +1,4 @@
+<?jelly escape-by-default='true'?>
+<div>
+    TODO
```

Comment:
What do we need to do here?

Reply:
This was generated automatically from the starter template for a Jenkins plugin! I never found that I needed to edit this file but at the same time, I'm not sure if deleting it was the right thing to do.

Reply:
I found out that "This view is used to render the installed plugins page." So I'm keeping it and just adding in the div what the plugin is.